### PR TITLE
Add config vars for perfect message and gerrit comments

### PR DESCRIPTION
**Pull request by dopuskh3**
Add the ability to configure message added by sputnik in gerrit for both comment and the former "Perfect !" message

### CODE REVIEW

File: src/main/java/pl/touk/sputnik/review/Review.java [hunk 1]
```diff
@@ -80,7 +81,9 @@ public void addError(String source, Violation violation) {
     }
 
     private void addError(@NotNull ReviewFile reviewFile, @NotNull String source, int line, @Nullable String message, Severity severity) {
-        reviewFile.getComments().add(new Comment(line, String.format(COMMENT_FORMAT, source, severity, message)));
+        reviewFile.getComments().add(new Comment(line, MessageFormat.format(
+                ConfigurationHolder.instance().getProperty(GeneralOption.COMMENT_FORMAT),
+                source, severity, message)));
```
Comment (SpOOnman) [inline src/main/java/pl/touk/sputnik/review/Review.java]: ReviewFormatter would be great. DTO names are bad, I just wanted to share my concept with container objects.

**Comment by SpOOnman** [comment]
@dopuskh3 will you take care of an update or should I take over with ReviewFormatter proposed by @pjagielski ?

**Comment by SpOOnman** [comment]
@dopuskh3 thanks, we'll wait for that.

**Comment by dopuskh3** [comment]
Did you had time to have a look on this MR ?

File: src/main/java/pl/touk/sputnik/configuration/GeneralOption.java [hunk 2]
```diff
@@ -9,7 +9,10 @@
     PROCESS_TEST_FILES("global.processTestFiles", "Process test files?", "true"),
     MAX_NUMBER_OF_COMMENTS("global.maxNumberOfComments", "Maximum number of comments to submit", "0"),
     COMMENT_ONLY_CHANGED_LINES("global.commentOnlyChangedLines", "Comment only changed lines and context", "false"),
-
+    COMMENT_FORMAT("global.commentFormat", "Sputnik comment format. {0}: reporter, {1}: level, {2}: message",
```
Comment (SpOOnman) [inline src/main/java/pl/touk/sputnik/configuration/GeneralOption.java]: I'd like to have separate package for messages and formats. messages.commentFormat, messages.problemFormat would be a nice separation

File: src/main/java/pl/touk/sputnik/configuration/GeneralOption.java [hunk 3]
```diff
@@ -22,6 +25,7 @@
 
     SCORE_STRATEGY("score.strategy", "Score strategy: <NoScore|ScoreAlwaysPass|ScorePassIfEmpty|ScorePassIfNoErrors>", "ScoreAlwaysPass"),
     SCORE_PASSING_KEY("score.passingKey", "Score passing key", "Code-Review"),
+    SCORE_PASSING_COMMENT("score.passingComment", "Comment when no errors are found", "Perfect!"),
```
Comment (SpOOnman) [inline src/main/java/pl/touk/sputnik/configuration/GeneralOption.java]: It's not well documented (of course). Scoring is for gerrit comments on a patchset. score package fits better as options, while this option would fit better to messages package, like messages.perfectComment.

File: src/main/java/pl/touk/sputnik/engine/visitor/SummaryMessageVisitor.java [hunk 4]
```diff
@@ -22,7 +30,7 @@ private void addSummaryMessage(Review review) {
 
     private String getSummaryMessage(@NotNull Review review) {
         if (review.getTotalViolationCount() == 0) {
-            return PERFECT_MESSAGE;
+            return perfectMessage;
         }
         return String.format("Total %d violations found", review.getTotalViolationCount());
```
Comment (SpOOnman) [inline src/main/java/pl/touk/sputnik/engine/visitor/SummaryMessageVisitor.java]: Let's parametrize this too!

**Comment by SpOOnman** [comment]
I am sorry that it took so long. I am now determined to care more about Sputnik.

**Pull request merged**